### DISCUSSION TITLE
Customize Nginx configuration will overwrite apisix confg？

### DISCUSSION THREAD

**Discussion by Lewisyixin** [apache/apisix#7543] [Q&A]
Our apisix is behind a slb with health check. I just want to disable the health check log only.
I see example in this link https://apisix.apache.org/docs/apisix/customize-nginx-configuration/
I add config below and found all route config on apisix not work, seems overwrite by this config. Is this normal?
```
  http_configuration_snippet: |
      server
      {
          listen 80;
          server_name check.com;
          access_log off;

          location /health {
                return 200 ok;
          }
      }
```

**Comment by tzssangglass** [comment]
You can look at the final generated nginx.conf to determine the problem.
